code/__HELPERS/names.dm

```diff
@@ -217,9 +215,36 @@ var/syndicate_code_response//Code response for traitors.
 						code_phrase += pick(adjectives)
 					if(3)
 						code_phrase += pick(verbs)
-		if(words==1)
-			code_phrase += "."
-		else
-			code_phrase += ", "
 
 	return code_phrase
+
+//Phrases are green, responses are blue.
+/proc/parse_for_code_words(text)
+	var/parse = text
+	for(var/word in syndicate_code_phrase)
+		var/indexl = 0
+		var/indexu = 0
+		var/parse1
+		var/parse2
+		var/insertword = "<span class='codephrase'>[word]</span>"
+		if(!findtext(parse, word))
+			continue
+		indexl = findtext(parse, word)

```

Comment:
"Look at my cat."

I'd say using \b would be better but this is pretty performance sensitive, and i'm not sure how byond regex efficiency goes.